server/src/immich/main.ts

```diff
@@ -29,20 +30,23 @@ export async function bootstrap() {
 
   const excludePaths = ['/.well-known/immich', '/custom.css'];
   app.setGlobalPrefix('api', { exclude: excludePaths });
-  // copied from https://github.com/sveltejs/kit/blob/679b5989fe62e3964b9a73b712d7b41831aa1f07/packages/adapter-node/src/handler.js#L46
-  // provides serving of precompressed assets and caching of immutable assets
-  app.use(
-    sirv('www', {
-      etag: true,
-      gzip: true,
-      brotli: true,
-      setHeaders: (res, pathname) => {
-        if (pathname.startsWith(`/_app/immutable`) && res.statusCode === 200) {
-          res.setHeader('cache-control', 'public,max-age=31536000,immutable');
-        }
-      },
-    }),
-  );
+  const staticDir = 'www';
```

Comment:
this is a relative path, can you use an absolute path instead? Maybe you can just rework this (remove the index.html part from the env)

```ts
export const WEB_ROOT_PATH = join(process.env.IMMICH_WEB_ROOT || '/usr/src/app/www', 'index.html');
```

Reply:
done